Accompany/Controllers/Core/Home/AddNoteDetailsTableViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  SemesterDetailsTableViewController.swift
+//  Accompany
+//
+//  Created by Kelbin David on 2022-02-10.
+//
+
+import UIKit
+import SnapKit
+
+protocol AddNoteDetailsTableViewControllerDelegate {
+  func add(todo: ToDo)
+  func edit(todo: ToDo)
+}
+
+class AddNoteDetailsTableViewController: UITableViewController {
```

Comment:
Please refactor the name of this file too.
```suggestion
class TodoFormTableViewController: UITableViewController {
```